node/forest_libp2p/src/service.rs

```diff
@@ -171,11 +197,47 @@ impl Libp2pService {
                         }
                         ForestBehaviourEvent::BlockSyncResponse { request_id, response, .. } => {
                             debug!("Received blocksync response (id: {:?}): {:?}", request_id, response);
-                            self.network_sender_out.send(NetworkEvent::BlockSyncResponse {
-                                request_id,
-                                response,
-                            }).await;
+                            let tx = self.bs_request_table.remove(&request_id);
+
+                            if let Some(tx) = tx {
+                                if let Err(e) = tx.send(response) {
+                                    debug!("RPCResponse receive failed: {:?}", e)
+                              }
```

Comment:
can you just fix the spacing on this? Got screwed up with last commit

Reply:
lol damn macros